### ISSUE TITLE
EnderIO fluid pipes doesn't connect

### ISSUE THREAD

**Issue by Stabington** [SlimeKnights/TinkersConstruct#2181]
They won't connect to neither the drains, nor the casting basin/tables.
They used to work in the previous version.

Mantle 0.12.2
Tinkers' 2.3.2
Forge 12.17.0.1965

**Comment by vdrake77** [comment]
Nor can you use the drain to pour into enderio tanks.  And there's no way to connect Tinker tanks to enderio fluid conduits either.

Are the contents of the smeltery no longer 'fluids'?  Or is this something to do with the new fluid capabilities mentioned in the update?

**Comment by KnightMiner** [comment]
This is due to the new fluid capabilities, which unfortunately are not compatible with mods using the old system (see also ForestryMC/ForestryMC#1308). It should work again once Ender IO updates to the new system.

**Comment by SoraHjort** [comment]
Since it looks like it'll be quite a while before Ender IO (and others such as NeoTech) implement the new Fluid system, is it possible to do what Forestry and a few other mods did, and reimplement the old system right next to the new system?

**Comment by bonii-xx** [comment]
Many things are possible, but we wont. Better hurry them to update :P

**Comment by SoraHjort** [comment]
People have tried, and it's just caused the EnderIO devs to be frustrated/annoyed since they have a lot on their plate. As seen here https://github.com/SleepyTrousers/EnderIO/issues/3375#issuecomment-225389466

The only issue I'm having with the new system is the mods that have pipes for fluid transfers haven't updated to it. Which makes mass automated production of the smeltery hard to do. Especially with the lack of the casting channel. If the Casting Channel was reimplemented then I'd have a work around.

**Comment by KnightMiner** [comment]
Casting channel is on the to do list, but the 1.7 version had a lot of issues so it's going to need a complete rewrite

**Comment by bonii-xx** [comment]
>People have tried, and it's just caused the EnderIO devs to be frustrated/annoyed since they have a lot on their plate.

And what makes you think we have any less to do, that we would have time to reimplement an outdated and deprecated API? :P

**Comment by SoraHjort** [comment]
Sure, redstone and faucets do work, but the issue comes with two (or more) casting basins have a single ingot between them all. There currently isn't a way to force them to be dumped into a single container automatically. The only option using just TC you have in regards to that is to use a seared tank to manually drain things out.

Anyway, for those who are looking for ways to currently do some form of mass processing, with the lack of any pipe mods being updated currently. I did come up with a workaround using multiple Fancy Fluid Storage tanks, a bunch of EnderIO's insulated redstone conduits, two RFTool's Timer, many of hoppers, and Hopperducts. And I screenshot them, though I probably should have turned down the FOV when I did. Oh well. http://imgur.com/a/HzMvC

It's certainly more complex, time consuming, resource intense, requiring a large space, and highly prone to clogging (if you put in more than one type of material), than if we were to have Casting Channels (kidding, take your time :P)
